biomedical_schema/BioOntologySchemaEnum.py

```diff
@@ -0,0 +1,6100 @@
+# This schema file is generated by parse_ebi.py. Please don't edit.
```

Comment:
Should this file be called BioOntologySchemaEnum.mcf (instead of .py)?

Also, does it make sense to give it a more specific name than "BioOntology"?

Reply:
Good call! I changed the name to MolecularInteractionsControlledVocabularyEnum.mcf. I'm changing this in the script that generated it as well and will send a PR shortly.